### ISSUE TITLE
[package] openssl/ALL: Missing Option "no_hw"

### ISSUE THREAD

**Issue by tschabo** [conan-io/conan-center-index#686]
At the moment it is not possible to use the option "no_hw".
Can we please add this option.

**Comment by danimtb** [comment]
@tschabo thanks for the request! Could you give us more context? Why are you using it? Why it is important? Anything else to consider regarding the option? Thanks!

**Comment by tschabo** [comment]
@danimtb If no-hw is not set, openssl tries to load certain dlls. These are searched for in the paths from the PATH environment variable. If a network path from PATH is temporarily unavailable, it may take minutes for openssl to initialize.

**Comment by danimtb** [comment]
Thanks, we will need to investigate this flag. Or in case you have it working already, you can contribute the feature in a pull request to the openssl recipe 😄

**Comment by SSE4** [comment]
seems like option is deprecated: https://github.com/openssl/openssl/pull/8380
in general, we need to review/cleanup the options for OpenSSL, together with #108

**Comment by tschabo** [comment]
@SSE4 You're right, but we are currently stuck with version 1.0.2 due to dependencies.

**Comment by Croydon** [comment]
The requested build option is deprecated upstream in more recent OpenSSL versions. The older versions are outdated. Therefore this issue can be closed
